cmplxs/cmplxs_test.go

```diff
@@ -725,6 +725,62 @@ func TestMul(t *testing.T) {
 	}
 }
 
+func TestMulConj(t *testing.T) {
+	s1 := []complex128{1 + 1i, 2 + 2i, 3 + 3i}
```

Comment:
Unlikely to happen but I'd use input data with differing real and imag parts in case the functions somehow mix them up. Here and in the other test cases.

Reply:
Less likely here than in `Scale*` will do it in both places.